EVENTS.md

```diff
@@ -36,6 +36,9 @@
 | admin_user_authenticate_after | 1.9.4.5 |
 | admin_user_authenticate_before | 1.9.4.5 |
 | admin_user_validate | 1.9.4.5 |
+| adminhtml_[entityCode]_attribute_types | 22.0.0 |
+| adminhtml_[entityCode]_attribute_scopes | 22.0.0 |
+| adminhtml_[entityCode]_attribute_edit_prepare_form | 22.0.0 |
```

Comment:
I'd use the next v20 version number for this, if it won't me merged in time we'll re-change it later, otherwise we can't merge with 22.0.0 since it doesn't exist.

Reply:
Updated to 20.0.15, although I think flyingmana said he's going to do a release this week so it might need to be updated again.